templates/workloads/services/worker/manifest.yml

```diff
@@ -27,6 +27,9 @@ image:
 
 cpu: {{.CPU}}       # Number of CPU units for the task.
 memory: {{.Memory}}    # Amount of memory in MiB used by the task.
+{{- if .Platform}}
```

Comment:
Will this be evaluated to true if `.Platform == ""`?

Reply:
nope!